models/cryptopunks/ethereum/cryptopunks_ethereum_events.sql

```diff
@@ -17,11 +13,11 @@ with cryptopunks_bids_and_sales as (
     from 
     (
     select  "PunkBought" as event_type
-            , `punkIndex` as punk_id 
-            , `value`/1e18 as sale_price   
+            , punkIndex as punk_id 
+            , value/1e18 as sale_price   
             , NULL as bid_amount 
```

Comment:
you may want to explicitly cast all these hardcoded null values to desired datatypes, similar to other models you have

Reply:
Makes sense - will update that now too!

Reply:
Updated!